--- a/backends/ForecaWeatherBackend.qml
+++ b/backends/ForecaWeatherBackend.qml
@@ -103,6 +103,8 @@
                 if (i % 3 !== 0) continue
                 weather.timestamp =  new Date(data.time)
                 weather.temperature = data.temperature
+                weather.windSpeed = data.windSpeed != null ? Math.round(data.windSpeed) : -1
+                weather.windDirection = data.windDir != null ? data.windDir : -1
             } else {
                 var dateArray = data.date.split("-")
                 weather.timestamp = new Date(parseInt(dateArray[0]),
