--- a/backends/OpenWeatherBackend.qml
+++ b/backends/OpenWeatherBackend.qml
@@ -93,6 +93,9 @@
                 weather.windDirection = data.wind.deg
                 weather.high = data.main.temp_max
                 weather.low = data.main.temp_min
+            } else {
+                weather.windSpeed = data.wind && data.wind.speed !== undefined ? Math.round(data.wind.speed) : -1
+                weather.windDirection = data.wind && data.wind.deg !== undefined ? data.wind.deg : -1
             }
             weatherData[weatherData.length] = weather
         }
