--- a/components/WeatherBanner.qml
+++ b/components/WeatherBanner.qml
@@ -217,6 +217,7 @@
                             height: dailyForecastList.height
                             DailyForecastItem {
                                 highlighted: weatherBanner.highlighted
+                                showWind: true
                                 onHeightChanged: if (model.index == 0) dailyForecastList.itemHeight = height
                             }
                         }
@@ -304,6 +305,7 @@
                             HourlyForecastItem {
                                 hourMode: hourlyForecastList.hourMode
                                 highlighted: weatherBanner.highlighted
+                                showWind: true
                                 onHeightChanged: if (model.index == 0) hourlyForecastList.itemHeight = height
                             }
                         }
