--- a/components/WeatherBanner.qml
+++ b/components/WeatherBanner.qml
@@ -213,6 +213,9 @@
                         }
 
                         delegate: Item {
+                            // sailfish-weather-wind: the forecast item shows wind lines when its parent has this
+                            property bool showWind: true
+
                             width: dailyForecastList.itemWidth
                             height: dailyForecastList.height
                             DailyForecastItem {
@@ -298,6 +301,9 @@
                         }
 
                         delegate: Item {
+                            // sailfish-weather-wind: the forecast item shows wind lines when its parent has this
+                            property bool showWind: true
+
                             width: hourlyForecastList.itemWidth
                             height: hourlyForecastList.height
 
